tests/spec/core/figures-spec.js

```diff
@@ -81,4 +81,63 @@ describe("Core - Figures", function() {
     expect(figLinks.item(0).textContent).toEqual("Figure 1 PREFIG");
     expect(figLinks.item(1).textContent).toEqual("Figure 2 IMGTIT");
   });
+
+  it("adds natural width and height to images if not set", async () => {
```

Comment:
let's change this do `describe("normalize images", async() => {})`. You need to check if `describe` takes an async function or not. I can't recall. If not, then create the `doc` in `beforeAll()`. 

Then inside the `describe()`,  wrap each test with an `it("checks something", async ()=>{})`